control_plane/src/local_env.rs

```diff
@@ -392,6 +399,36 @@ impl LocalEnv {
         }
     }
 
+    /// Inspect the base data directory and extract the instance id and instance directory path
+    /// for all storage controller instances
+    pub async fn get_storage_controller_instances(&self) -> std::io::Result<Vec<(u8, PathBuf)>> {
+        let mut instances = Vec::default();
+
+        let dir = std::fs::read_dir(self.base_data_dir.clone())?;
```

Comment:
do we not use camino in control_plane at all? it'd have utf8 versions. no need to fix up this one place if camino migration is done at all, but just FYI for the next place.

Reply:
We do, but it's patchy. `LocalEnv::base_data_dir` is not a utf8 path buf and I'd rather not change in this PR.